src/compartment.c

```diff
@@ -245,7 +245,7 @@ comp_from_elf(char* filename)
     }
 
     comp_register_ddc(new_comp);
-    comp_print(new_comp);
+    /*comp_print(new_comp);*/
```

Comment:
Commented code.

Reply:
Fixed in https://github.com/capablevms/CHERI-ELF-comp/pull/3/commits/00073d1d86c091d037fc9ba6e931856eda0333fd.